MultiPage-Clone/main.js

```diff
@@ -1,4 +1,14 @@
+const burgerr = document.querySelector('.navimage');
+const nav = document.querySelector('.nav-links');
+const body = document.querySelector('body');
 
-const OVERFLOW = $('#toggle1');
+burgerr.addEventListener('click', () => {
```

Comment:
Right now we just reload a page on click. No javascript is used basically.
Is there a requirement for the task to reload only a part of the page as in the [demo](https://www.wix.com/website-template/view/html/2622)?